cirq-core/cirq/ops/gateset.py

```diff
@@ -76,10 +91,17 @@ def __init__(
             description: Human readable description of the gate family.
             ignore_global_phase: If True, value equality is checked via
                 `cirq.equal_up_to_global_phase`.
+            tags_to_accept: If non-empty, only TaggedOperations containing at least one tag in this
+                sequence can be accepted. This parameter cannot contain tags in `tags_to_ignore`.
+            tags_to_ignore: A TaggedOperation is rejected if it contains at least one tag in
+                this sequence. If a TaggedOperation contains tags in both `tags_to_accept` and
+                `tags_to_ignore`, it is rejected. This parameter cannot contain tags in
+                `tags_to_accept`.
```

Comment:
Use `cirq.Operation` instead of TaggedOperation; the former also has a `op.tags` field and there are a few open issues for discussion to try and deprecate the tagged operation class in favour of moving the .tags field entirely on operations.